NEO-TAT/tat_flutter
# feat: add "Themed Icons" for Android

## Description

This PR adds support for [Themed Icons](https://developer.android.com/about/versions/13/features?hl=zh-tw#themed-app-icons) on Android. This feature allows users running Android 13 or later theme our app icon based on the system theme.

## Implementation

- Add monochrome icon PNGs for mdpi, hdpi, xhdpi, xxhdpi and xxxhdpi.
- Add `<monochrome>` element in `ic_launcher.xml` to enable this feature.

## Testing Instructions

Install the app on Android, drag a shortcut onto the home screen, and enable "Themed Icons" in the launcher settings. Verify that the TAT logo correctly follows the system theme.

## Additional Notes

N/A.
